docs/config/README.md

```diff
@@ -890,6 +891,67 @@ The module will be shown if any of the following conditions are met:
 format = "via [e $version](bold red) "
 ```
 
+## Gcloud
+
+The `gcloud` module shows the current configuration for [`gcloud`](https://cloud.google.com/sdk/gcloud) CLI.
+This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gcloud/configurations/config_{CONFIG NAME}` file and the `CLOUDSDK_CONFIG` env var.
+
+### Options
+
+| Variable          | Default                                           | Description                                                                 |
+| ----------------- | ------------------------------------------------- | --------------------------------------------------------------------------- |
+| `format`          | `"on [$symbol$account(\\($region\\))]($style) "`  | The format for the module.                                                  |
+| `symbol`          | `"🇬️ "`                                            | The symbol used before displaying the current GCP profile.                  |
```

Comment:
As mentioned in the linked issue, given AWS and gcloud have very different version #s I don't think using ☁️ in both would cause too much confusion

Reply:
Agreed. I've replaced 🇬️ with ☁️ .